public/app/services/settingsService.js

```diff
@@ -13,17 +13,22 @@ angular.module('kibibitCodeEditor')
     var isFullscreen = false;
     var currentUndoManager = undefined;
     var currentEditor = undefined;
+    var modelist = ace.require('ace/ext/modelist');
+    var themelist = ace.require('ace/ext/themelist');
+    var editorSettings = new EditorSettings();
 
     /* EXPOSE SIMPLE VARS */
     // This are probably better off in state instead of settings. but they're here for now :-)
     settings.cursor = cursor;
     settings.currentUndoManager = currentUndoManager;
     settings.currentEditor = currentEditor;
+    settings.modelist = modelist;
```

Comment:
now, because my other comment (about the constants), you don't have to put those two on the settings service.
import the constants and use them here.

to import constants, just do:

```javascript
.service('SettingsService', ['EDITOR', function(EDITOR) {
```

I'm not sure if `EDITOR` is a good name. if you find a better one, go ahead 😉